bin.src/generateCoefficients.py

```diff
@@ -34,11 +35,11 @@
 
     # Parse orbit file input values.
     if args.orbitFile is None:
-        print "Must specify orbit file to use."
+        print("Must specify orbit file to use.")
         exit()
 
     if not os.path.isfile(args.orbitFile):
-        print "Could not find orbit file %s" %(args.orbitFile)
+        print("Could not find orbit file %s" % (args.orbitFile))
     if args.orbitFile.lower().endswith('s3m'):
         skiprows = 1
```

Comment:
Why not just specify a character indicating "this line is a comment"?  In the S3M files, I *think* the header line starts with "!!"